end-to-end/cypress/e2e/portal/pages/publication.cy.js

```diff
@@ -47,8 +47,8 @@ describe("Publication page", () => {
     });
     it("has a publication date, which is labeled as a preprint date", () => {
```

Comment:
```suggestion
    it("has a publication date", () => {
```
since it's also no longer a preprint 😄

Reply:
Good catch!